flixel/FlxCamera.hx

```diff
@@ -1594,17 +1597,20 @@ class FlxCamera extends FlxBasic
 	/**
 	 * Sets the filter array to be applied to the camera.
 	 */
+	@:deprecated("setFilters() is deprecated, use the filters array instead")
 	public function setFilters(filters:Array<BitmapFilter>):Void
 	{
-		_filters = filters;
+		this.filters = filters;
 	}
 
 	/**
 	 * Gets the filter array applied to the camera.
+     * WARNING: can return `null`.
 	 */
-	public function getFilters()
+	@:deprecated("getFilters() is deprecated, use the filters array instead")
+	public function getFilters():Array<BitmapFilter>
 	{
-		return _filters;
+		return filters;
```

Comment:
since this was just added last week in #2888 and was never released, let's just remove it. @Vortex2Oblivion, be sure to update your projects that use this

Also, in the future, rather than warning them about null we should just use Null<T> return types

Reply:
Yeah fair